src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,22 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    <ul>
+      {todos.map((todo: Todo) => (
```

Comment:
```suggestion
      {todos.map((todo) => (
```
You don't have to type it here. TypeScript knows it's of type Todo as you've specified it earlier (in your Props).